application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -21,24 +25,28 @@
 public final class TagCommand extends SlashCommandAdapter {
     private final TagSystem tagSystem;
 
+    private static final String COMMAND_NAME = "tag";
     static final String ID_OPTION = "id";
     static final String REPLY_TO_USER_OPTION = "reply-to";
+    private static final int MAX_OPTIONS = 25;
```

Comment:
OptionData#MAX_OPTIONS or OptionData#MAX_CHOICES

Not sure what it's called

Reply:
MAX_CHOICES it is